src/sentry/api/client.py

```diff
@@ -60,11 +60,13 @@ def request(self, method, path, user=None, auth=None, params=None, data=None,
                 mock_request.is_superuser = lambda: request.is_superuser()
             else:
                 mock_request.is_superuser = lambda: is_superuser
+            mock_request.session = request.session
         else:
             mock_request.auth = auth
             mock_request.user = user
             mock_request.is_sudo = lambda: is_sudo
             mock_request.is_superuser = lambda: is_superuser
+            mock_request.session = {}
```

Comment:
This will definitely cause problems though, doing like, `request.session.modified = True` if the branch is ever hit.